sentences/fi/light_HassTurnOn.yaml

```diff
@@ -3,7 +3,7 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "<turn> <area> (valo | valot) päälle"
+          - "<päälle> [kaikki] {area} <valo> <päälle>"
```

Comment:
I think we need to be a bit more careful with these. This sentence only triggers when it starts with `<päälle>` and ends with it too.

`laita päälle olohuone valo laita päälle` does not sound too natural.

Rough translation: "turn on livingroom light turn on"